sematic/db/models/json_encodable_mixin.py

```diff
@@ -16,41 +16,37 @@ class JSONEncodableMixin:
     Defines JSON-encodable behavior.
     """
 
-    def to_json_encodable(self):
+    def to_json_encodable(self, redact: bool = True):
```

Comment:
I think a default of `False` makes more sense from a "principle of least surprise" perspective:

```
# I would be surprised if this doesn't pass
assert original == Foo.from_json_encodable(original.to_json_encodable())
```

```
# I would not be surprised if this fails--the redact=True indicates why it doesn't
assert original == Foo.from_json_encodable(original.to_json_encodable(redact=True))
```

Reply:
I agree with @tscurtu, from a security standpoint it seems more rigorous to operate on need-to-know basis.

Reply:
Maybe a longer term solution is to leave model-level payloads unredacted so that this works
```python
assert original == Foo.from_json_encodable(original.to_json_encodable())
```
and enforce redaction in `payloads.py` instead.

For now, I'll leave the default to `True`.